index.html

```diff
@@ -7714,6 +7714,80 @@ <h3>Processing actions</h3>
  input JSON, such that the actions to be performed in a single <a>tick</a>
  are grouped together.
 
+<p>An <dfn>element origin</dfn> is a Map
+which [=map/exist|contains=] "<code>type</code>" and
+"<code>sharedId</code>", and for which the value of the entry with key
+"<code>type</code>" is "<code>element</code>".
```

Comment:
In the BiDi spec we have `type=node` when serializing DOM nodes. So I assume that we should have that here as well?

Reply:
Yeah, although given this should really be an Element and not something else, I wonder if we should actually give Element specific serialization that differs from Node. I'll file an issue.